test/views/CheckYourAnswersViewSpec.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import controllers.routes
+import models.{C79Certificate, DutyDefermentStatement, FileRole, NormalMode}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.scalatest.Assertion
+import viewmodels.CheckYourAnswersHelper
+import views.html.CheckYourAnswersView
+
+class CheckYourAnswersViewSpec extends ViewTestHelper {
+  "view" should {
+    "display correct title and contents" when {
+      "the file role is C79 Certificate" in new Setup {
+        implicit val viewDoc: Document = view()
+
+        titleShouldBeCorrect(viewDoc, "cf.historic.document.request.review.C79Certificate.title")
+        pageShouldContainBackLinkUrl(viewDoc, returnUrl)
+        shouldContainSelectedDuration
+        shouldContainChangeLink
+        shouldContainConfirmAndSendButton
+      }
+
+      "the file role is Duty Deferment" in new Setup {
+        implicit val viewDoc: Document = view(DutyDefermentStatement)
+        titleShouldBeCorrect(viewDoc,
+          "cf.historic.document.request.review.DutyDefermentStatement.title")
+
+        shouldContainSecondaryHeading(view = viewDoc, accountNumberText =
+          s"${msg("cf.account.detail.requested.deferment-account-secondary-heading")} accountNumber")
+      }
+
+      "the file role is Duty Deferment and NI Indicator is enabled" in new Setup {
+        implicit val viewDoc: Document = view(DutyDefermentStatement, Some(true))
+        titleShouldBeCorrect(viewDoc,
+          "cf.historic.document.request.review.DutyDefermentStatement.title")
+
+        shouldContainSecondaryHeading(view = viewDoc, accountNumberText =
+          msg("cf.account.detail.requested.deferment-account-secondary-heading.NiAccount"))
+      }
+    }
+
+    trait Setup {
```

Comment:
We typically put the Setup last in order to be consistent.

Reply:
I've now moved them for all the specs i've worked on.